setup/quickstart/faq.md

```diff
@@ -9,6 +9,25 @@ sidebar:
 
 Here are a few Q/A pairs that come up fairly frequently.
 
+## I'm unable to load the Applications screen
+
+After installing Spinnaker and navigating to the <em>Applications</em> screen, you may see one of
+following issues:
+ * The loading indicator spins continuously (prior to release 1.9)
+ * The following error message is displayed: <em>Error fetching applications</em> (release 1.9 and later)
+
+ The most common cause of this error is that your browser cannot communicate with your Gate endpoint.
+ (This endpoint defaults to `http://localhost:8084`, but can be customized.)
+
+ Check your browser console log and/or network for any failed requests to `<gate-endpoint>/applications`.
+ Some things to check while troubleshooting:
+ * If you are accessing Spinnaker via the default `http://localhost:9000`, check that you have
+   forwarded Gate's port (8084 by default) to the machine where your browser is running.
```

Comment:
Can we provide instructions to check? 

Since `hal deploy connect` does this automatically, under what circumstances would this happen?

Reply:
I think this would mostly be if you ran `hal deploy connect` from some machine in the cloud but only forwarded port 9000 (but not 8084) to your local machine.  There are enough possible set-ups here that I'm not sure exactly how to ask people to further check.